app/assets/stylesheets/_typography.scss

```diff
@@ -59,7 +59,11 @@ pre {
   padding: 15px;
 }
 
-textarea { overflow: auto; }
+textarea {
+  min-height: 1em; // ensures at least one few lines of text show up to start
+  // max-height: 8em;
+  width: 100%; //text area fills up the full width of the content area
```

Comment:
Can we actually pull out these comments? A good rule of thumb is that the code should be able to speak for itself, and I think this does :)
🐘

Reply:
I think comments can have value in providing extra context for certain complex bits of code, but I agree with you here that these ones aren't really necessary. Also If the max height isn't needed anymore then can we delete it please rather than commenting it out?